tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Controllers/OrganisationControllerTests.cs

```diff
@@ -410,5 +410,231 @@ public static async Task GetServiceRecipientsAsync_OrganisationExists_ReturnsThe
 
             response.Should().BeEquivalentTo(new ActionResult<List<ServiceRecipientsModel>>(expected), config => config.WithStrictOrdering());
         }
+
+        [Test]
+        public static async Task GetOrganisationsAsync_OrganisationExists_ReturnsOrganisationAndRelatedOrganisation()
+        {
+            Guid relatedOrganisationId = Guid.NewGuid();
```

Comment:
In contrast to my earlier `new()` comment, I think `var` would be fine here. It's fairly obviously that it's going to be a `Guid`. 😃

(Same applies to the similar line in other tests.)